src/main/java/com/exasol/errorcodecrawlermavenplugin/config/ErrorCodeConfigException.java

```diff
@@ -1,7 +1,7 @@
 package com.exasol.errorcodecrawlermavenplugin.config;
 
 /**
- * Exception that is thrown on an invalid errorCodeConfig.yml
+ * Exception that is thrown on an invalid error_code_config.yml
```

Comment:
I would remove the file name from the Javadoc, so you don't need to touch the comments when you rename/ change the file (as it happened now)

Reply:
The only place I would leave the file name is the constant CONFIG_NAME